admin/app/controllers/DeploysRadiatorController.scala

```diff
@@ -16,8 +16,12 @@ trait DeploysRadiatorController extends Controller with Logging with Requests {
   val riffRaff: RiffRaffService
   val teamcity: TeamcityService
 
-  def getDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
-    riffRaff.getRiffRaffDeploys(pageSize, projectName, stage).map(ApiResults(_))
+  def getCompletedDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
```

Comment:
In theory yes, I didn't want to change to much while I am still experimenting. I am happy to reconsider the `projectName` argument if/when I cleanup